app/Repositories/BaseRepository.php

```diff
@@ -0,0 +1,334 @@
+<?php
+namespace App\Repositories;
+
+use Illuminate\Support\Facades\DB;
+use App\Repositories\RepositoryInterface;
+
+abstract class BaseRepository implements RepositoryInterface
+{
+
+    /**
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/BaseRepository.php</br>**Line**: 141</br>Whitespace found at end of line